flink-python/src/test/java/org/apache/flink/table/runtime/typeutils/serializers/python/timestamp/TimestampSerializerTest.java

```diff
@@ -16,10 +16,11 @@
  * limitations under the License.
  */
 
-package org.apache.flink.table.runtime.typeutils.serializers.python;
+package org.apache.flink.table.runtime.typeutils.serializers.python.timestamp;
```

Comment:
This doesn't seem correct. shouldn't it be in the same package as the serializer?

Reply:
Good catch.
Looks like an artifact after merge (since I started it before `SerializerTestBase` issue was solved) 
Now I completely removed all changes for this class from this PR